modules/hub/pages/hub-api-namespaces.adoc

```diff
@@ -0,0 +1,26 @@
+[[hub-api-namespaces]]
+= Hub XMRPC API namespaces
+
+
+Hub supports 3 different namespaces.
+
+
+[[name-spaces]]
+[cols="0,1,1", options="header"]
+.Namespaces
+|===
+| Name                  | Description  | Example
+| hub                   | to target the Hub itself. | `client.hub.login(login, password)`
```

Comment:
Please expand the descriptions a bit (if they become too long, you might even consider not using a table).

Eg. "in this namespace are some Hub-specific XMLRPC API endpoints, generally not available on peripheral Servers. For a list of those endpoints see..."

or

"this namespace redirects any call transparently to one specific Server and returns back any return value as if the Server's XMLRPC API endpoint was used directly. Use this namespace to operate on an individual Server (including any client or {productname} Proxy attached to it)"